pkg/framework/aries/default.go

```diff
@@ -43,13 +49,21 @@ func defFrameworkOpts(frameworkOpts *Aries) error {
 		frameworkOpts.outboundTransports = append(frameworkOpts.outboundTransports, outbound)
 	}
 
-	if frameworkOpts.storeProvider == nil {
-		storeProv, err := storeProvider()
+	currentStoreProvider := frameworkOpts.storeProvider
+	if currentStoreProvider == nil {
+		var err error
+		currentStoreProvider, err = storeProvider()
+
```

Comment:
Please, revert this change. It does nothing.
The line below will look like:
`err := assignVerifiableStoreIfNeeded(frameworkOpts, frameworkOpts.storeProvider)`

Reply:
Fixed